src/sprites/setup.py

```diff
@@ -0,0 +1,241 @@
+                     self._direction_exceptions.values(),
+                     self._exceptions.values(),
+                     [self.default]]:
+            for hitbox in item:
+                hitbox.update(
+                    hitbox.left * factor,
+                    hitbox.top * factor,
+                    hitbox.width * factor,
+                    hitbox.height * factor
+                )
+
+
+def state_importer(
+        path: str, size: int, state: EntityState, directions: list[Direction],
+        hitbox: _Hitbox
+) -> dict[Direction, _AniFrames]:
+
+    directions_dict = {}
+    full_path = os.path.join(path)
+    surf = pygame.image.load(full_path).convert_alpha()
+
+    for row, direction in enumerate(directions):
+        frames = []
+
+        for col in range(surf.get_width() // size):
+            subsurface = surf.subsurface(
+                col * size,
+                row * size,
+                size,
+                size
+            )
+            subsurface = pygame.transform.scale_by(subsurface, SCALE_FACTOR)
+            frames.append(subsurface)
+
+        current_hitbox = hitbox.get_hitbox(state, direction)
+        directions_dict[direction] = _AniFrames(
+            frames, current_hitbox
+        )
+
+    if Direction.LEFT in directions and Direction.RIGHT not in directions:
+        frames = []
+        for i in range(len(directions_dict[Direction.LEFT])):
+            frame = directions_dict[Direction.LEFT].get_frame(i)
+            frame = pygame.transform.flip(frame, True, False)
+            frames.append(frame)
+        hitbox = hitbox.get_hitbox(state, Direction.RIGHT)
+        directions_dict[Direction.RIGHT] = _AniFrames(frames, hitbox)
+    elif Direction.RIGHT in directions and Direction.LEFT not in directions:
+        frames = []
+        for i in range(len(directions_dict[Direction.RIGHT])):
+            frame = directions_dict[Direction.RIGHT].get_frame(i)
+            frame = pygame.transform.flip(frame, True, False)
+            frames.append(frame)
+        hitbox = hitbox.get_hitbox(state, Direction.LEFT)
+        directions_dict[Direction.LEFT] = _AniFrames(frames, hitbox)
+    return directions_dict
+
+
+def entity_importer(
+        path: str, size: int, directions: list[Direction],
+        hitbox: _Hitbox
+) -> dict[EntityState, dict[Direction, _AniFrames]]:
+    hitbox.scale_hitboxes(SCALE_FACTOR)
+    states = {}
+    for folder_path, sub_folders, file_names in os.walk(path):
+        for file_name in file_names:
+            current_state = EntityState(file_name.split(".")[0])
+            states[current_state] = state_importer(
+                path=os.path.join(folder_path, file_name),
+                size=size,
+                state=current_state,
+                directions=directions,
+                hitbox=hitbox,
+            )
+    return states
+
+
+class EntityAssets:
+    CHICKEN: EntityAsset
+    COW: EntityAsset
+    RABBIT: EntityAsset
```

Comment:
You can also specify the type of a namespace attribute when defining it. And don't worry, you can always replace the value later

```suggestion
from types import SimpleNamespace

ENTITY_ASSETS = SimpleNamespace()

ENTITY_ASSETS.COW: EntityAsset | None = None
ENTITY_ASSETS.CHICKEN: EntityAsset | None = None
ENTITY_ASSETS.RABBIT: EntityAsset | None = None
```